### ISSUE TITLE
The idea of ensemble in this model

### ISSUE THREAD

**Issue by wanyueli** [DS3Lab/RosENet#11]
Hi！In your paper you said that "Each combination of features and network architecture was initially trained with the PDBBind refined set minus the core set for 300 epochs. We trained 20 replicas from scratch, and for each replica, the model that minimized the validation error of the core set was saved. " Does this means we should run this code from the preprocess step to the test step 20 times and then choose five models with the lowest error to get the final results?

**Comment by hhassan1** [comment]
Hi @wanyueli , you are mostly correct.

The only difference is that we only preprocessed the dataset once. The "scratch" part was meant to say that we ran the training 20 times, for each training saving the model on the iteration where the lowest validation error was achieved.

The ensambling was done manually, it is not part of the pipeline. For ensambling we saved the 5 best predictions (best according to the RMSE of the validation set) and averaged the prediction of each complex across these 5 predictions.
